docs/how-to/installation/docker.txt

```diff
@@ -170,6 +170,17 @@ Note that if an agent container is running on the same machine as the
 master container, you may use ``determined-master`` (or whatever the
 name of the master container is) as the ``DET_MASTER_HOST``.
 
+Determined internally makes use of `Fluent Bit
+<https://fluentbit.io>`__; the agent requires the
+``fluent/fluent-bit:1.5`` Docker image to be present at runtime.
+Normally, the agent will pull the Fluent Bit image automatically, but,
+if the machines in the cluster are not able to access Docker Hub, the
```

Comment:
fun grammar style question (i'm bad at this). I would've done
```suggestion
Normally, the agent will pull the Fluent Bit image automatically, but
if the machines in the cluster are not able to access Docker Hub the
```
since the if clause affects the meaning of what comes after it and isn't just additive.
Why did you pick this?

Reply:
The second comma is a pretty clear win for me. Not entirely mandatory, but correct (consider the standalone "if A, then B") and, as a matter of style, I think too much of the sentence runs together without it. I think you're right that the first one doesn't belong, though. Even I can't be perfect every time. :wink: